website_chd_product_configurator/static/src/js/website_chd_product_configurator.js

```diff
@@ -0,0 +1,103 @@
+$(document).ready(function () {
+	console.log(openerp);
+	$('.type_selection').each(function () {
+	    var type_selection = this;
+	    $(type_selection).on('click', '.type_selection', function ()
+	    {
+	    	openerp.jsonRpc('/chd_init/getch/', 'call',
+	    		 {
+	             'type_id': this.value,
+	    		  }).then(function (data) {
+	    			  			var  dd_list=document.getElementById("type_select_id");
+	    			  			var data_js = eval(data);
+	    			  			$('#fini_select_id').empty();
+	    			  			$('#fini_select_id').append("<option></option>");
+	    			  			for (var key in data_js)
+	    			  			{if (data_js.hasOwnProperty(key))
+	    			  			    {
+	    			  			        $('#fini_select_id').append("<option value='" +data_js[key].id+ "'>"+ data_js[key].name+ "</option>");
+
+	    			  			    }
+	    			  			}
+	    		  })
+	    });
+	});
+
+	$(':input').each(function() {
```

Comment:
Isn't this a bit dangerous? Think of a widget in some side bar that contains a newsletter subscription form.

I would give the form in my template an id and select everything here relative to this id